Documentation/intro.rst

```diff
@@ -0,0 +1,78 @@
+Introduction to Polycube
+========================
+
+Polycube brief
+--------------
+
+``Polycube`` is an **open source** software framework that provides **fast** and **lightweight** **network functions**. Already available services, called `cubes`, include `bridge`, `router`, `nat`, `load balancer`, `firewall`, `DDoS mitigator`, and more.
+
+Cube can be composed to build arbitrary **service chains** and provide custom network connectivity to **namespaces**, **containers**, **virtual machines**, and **physical hosts**.
+
+Cubes are extremely **efficient** because are based on the recent `BPF` and `XDP` Linux kernel technologies. In addition, cubes are easily **extensible** and **customizable**.
+
+Polycube networks can be controlled by a coherent, uniform and simple command line, named `polycubectl`, which interacts with a REST-based daemon, `polycubed`.
+
+Polycube provides also two proof-of-concept **standalone applications** based upon this framework.
+`Pcn-k8s` is a brand new network plug-in for *Kubernetes*, which can handle the network of an entire datacenter and that outperforms existing solutions in terms of throughput.
+`Pcn-iptables` is a more efficient and scalable clone of *iptables*.
```

Comment:
```suggestion
`PCN-iptables` is a more efficient and scalable clone of existing Linux *iptables*.
```

Reply:
Used 'lowercase' in the rest of the text; changed to 'pcn-iptables'.